Add support for WC polyfilled browsers,

tomalec opened the pull request:
Update
- Starcounter/starcounter-include@4.2.0
  - Juicy/imported-template@3.1.1
- webcomponents/webcomponentsjs@1.0.22
- Palindrom/palindrom-redirect@1.0.0

https://github.com/Starcounter/starcounter-include/issues/69

tomalec committed:
Add support for WC polyfilled browsers,

Update
- Starcounter/starcounter-include@4.1.0
  - Juicy/imported-template@3.1.1
- webcomponents/webcomponentsjs@1.0.22

https://github.com/Starcounter/starcounter-include/issues/69

tomalec committed:
Prepare for working with BlendingProvider

Unhardcode CompositionProvider

Update Starcounter/starcounter-include@4.2.0

warpech commented:
When https://github.com/Palindrom/palindrom-redirect/pull/9 is released and added here, I think it can be merged and released as SCF 2.1.0.

tomalec committed:
Upgrade Palindrom/palindrom-redirect@1.0.0 CEv1,

an **autonomous** element,

Problem: it was not working with latest WebComponents.js polyfill

warpech commented:
Setting as “Finished” because it is:
- released (with release notes) https://github.com/Starcounter/StarcounterClientFiles/releases/tag/2.1.1
- included in level1 https://github.com/Starcounter/level1/pull/4435
- announced on the Starcounter blog https://starcounter.io/starcounter-2-4-gets-upgrade-polymer-2/ ("Manual upgrade or downgrade to Polymer 1 and 2")
